recipes/monotonicnetworks/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge,defaults,pytorch
```

Comment:
This file isn't needed. Please remove it.

I see you added it to make windows build pass. As this is a noarch recipe, it will only be built on linux. It's ok if windows build fails in staged recipes.